src/MCPClient/lib/clientScripts/parse_mets_to_db.py

```diff
@@ -13,6 +13,8 @@
 import fileOperations
 import databaseFunctions
 
+import django
+django.setup()
```

Comment:
Does this need to happen before `fileOperations`/`databaseFunctions` is imported?  I usually moved the Django imports & init above the archivematicaCommon imports.